content/reference/admin/users/index.md

```diff
@@ -18,6 +18,10 @@ To invite a user to your organization, click on the **Invite** button and fill i
 - **Email address**: The email address to which the invitation will be sent. The invited user doesn't have to use the same address in their user profile once connected.
 - **Organization role**: either choose a global role, or select none and specify each role by team according to the permissions you want to grant.
 
+{{< alert warning >}}
+Only global system admin can generate an invitation
```

Comment:
One last review:
I think the following sentence is better worded:
"Only global system admins can generate invitations."